### PR TITLE
Adding an update to the readme

### PR DESCRIPTION
**Pull request by ekirkland**
This is my first PR!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# evan-test
\ No newline at end of file
+# evan-test
+
+This is my readme!
```
Comment (evankland) [inline README.md:3]: It's a nice readme, but should you add more?

**Review by evankland — COMMENTED** [review]
Just wondering if you should add more?

File: README.md (hunk 1 above)
Comment (ekirkland) [reply README.md:3]: There will be more to come later, this is just the start!

**Review by evankland — APPROVED** [review]
Cool, looking forward to more!

**Pull request merged**